src/main/java/com/eternalcode/discordapp/DiscordApp.java

```diff
@@ -61,6 +65,12 @@ public static void main(String... args) throws InterruptedException {
         configManager.load(databaseConfig);
         configManager.load(experienceConfig);
 
+        ConfigManager dataManager = new ConfigManager(new File("data"));
```

Comment:
Zrób to po przez resource w klasie konfiguracyjniej

Reply:
Tylko że to jest dataManager, jako parametr jest podany folder którym ma być **data** tak jak przy config

Reply:
https://github.com/EternalCodeTeam/DiscordOfficer/pull/89/files/a62a5e99ef57960a670795d7e0e54e3c27e820aa#diff-5f26324444a3fed321924c299c03e02307398c5e1b020f3d5c2fa44572e87e8aR38-R41

Reply:
> https://github.com/EternalCodeTeam/DiscordOfficer/pull/89/files/a62a5e99ef57960a670795d7e0e54e3c27e820aa#diff-5f26324444a3fed321924c299c03e02307398c5e1b020f3d5c2fa44572e87e8aR38-R41

Stwórz najpierw directory a w https://github.com/EternalCodeTeam/DiscordOfficer/pull/89/files/a62a5e99ef57960a670795d7e0e54e3c27e820aa#diff-5f26324444a3fed321924c299c03e02307398c5e1b020f3d5c2fa44572e87e8aR38-R41 odwołuj się do niego

```
new File("data")
new File("config)"

a później data + File.separator + plik.yml